notebooks/04_filter.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<p>This can be misleading because we actually filter <code>id=0_Parade_image_01</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Fix.